core-web/libs/portlets/dot-locales/portlet/src/lib/dot-locales-list/store/dot-locales-list.store.ts

```diff
@@ -16,64 +39,232 @@ export interface DotLocaleRow {
     actions: DotActionMenuItem[];
 }
 
-export interface DotLocalesListState {
+export interface DotLocalesListState extends DotLanguagesISO {
     status: ComponentStatus;
     locales: DotLocaleRow[];
+    initialLocales: DotLanguage[];
+    isEnterprise: boolean;
+    pushPublishEnvironments: DotEnvironment[];
 }
 
-export interface DotLocaleListViewModel {
+export interface DotLocaleListViewModel extends DotLanguagesISO {
     locales: DotLocaleRow[];
+    initialLocales: DotLanguage[];
+}
+
+export interface DotLocaleListResolverData {
+    localaes: DotLanguage[];
```

Comment:
you have a typo here and this interface looks like an extension of an interface you declared in another file

Reply:
removed